Makefile

```diff
@@ -287,6 +287,15 @@ golangci: get-ci-tools
 golint: get-ci-tools
 	PATH=$(GOBIN):$(PATH); $(CI_TOOLS_REPO_DIR)/test-runner/golint.sh
 
+# Run go mod tidy against code
+APIPATH ?= $(shell pwd)/api
+.PHONY: tidy
+tidy: fmt
```

Comment:
it should not depend on fmt, there were follow ups in other operators https://github.com/openstack-k8s-operators/placement-operator/pull/153

Reply:
Hey thanks for review. I have also made some changes as per the comment given here [1]

[1]: https://github.com/openstack-k8s-operators/cinder-operator/pull/147#discussion_r1158504341